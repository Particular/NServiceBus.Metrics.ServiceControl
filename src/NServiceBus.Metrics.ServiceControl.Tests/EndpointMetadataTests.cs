namespace NServiceBus.Metrics.ServiceControl.Tests
{
    using NServiceBus.Metrics.ServiceControl.ServiceControlReporting;
    using NUnit.Framework;
    using Particular.Approvals;

    [TestFixture]
    class EndpointMetadataTests
    {
        [Test]
        public void Verify_serialization()
        {
            var endpointMetadata = new EndpointMetadata("localAddress");
            var json = endpointMetadata.ToJson();

            Approver.Verify(json);
        }
    }
}
