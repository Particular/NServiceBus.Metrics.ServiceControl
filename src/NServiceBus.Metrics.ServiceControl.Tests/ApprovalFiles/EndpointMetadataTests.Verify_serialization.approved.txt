{"PluginVersion":3,"LocalAddress":"localAddress"}